rosbag2_cpp/src/rosbag2_cpp/writers/sequential_writer.cpp

```diff
@@ -354,6 +354,10 @@ void SequentialWriter::write(std::shared_ptr<const rosbag2_storage::SerializedBa
   const auto message_timestamp = std::chrono::time_point<std::chrono::high_resolution_clock>(
     std::chrono::nanoseconds(message->time_stamp));
 
+  if (!message_within_accepted_time_range(message_timestamp)) {
```

Comment:
Please update the description for the `write(..)` function in the header file according to the changed behavior.